src/free_addresses/free_addresses.go

```diff
@@ -0,0 +1,181 @@
+package free_addresses
+
+import (
+	"encoding/json"
+	"encoding/xml"
+	"fmt"
+	"net"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/ori-amizur/introspector/src/util"
+	"github.com/pkg/errors"
+
+	"github.com/filanov/bm-inventory/models"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	AddressLimit      = 8000
+	MinSubnetMaskSize = 22
+)
+
+//go:generate mockery -name IExecute -inpkg
+type IExecute interface {
```

Comment:
'I' prefix is not the golang convention.
See https://golang.org/doc/effective_go.html#interface-names

Reply:
OK